src/server/models/url.ts

```diff
@@ -164,7 +164,7 @@ export const Url = <UrlTypeStatic>sequelize.define(
         fields: ['userId'],
       },
       {
-        name: 'urls_weighted_search_idx',
+        name: 'new_urls_weighted_search_idx',
```

Comment:
please, never claim something is new. a more appropriate name could be `url_entry_weighted_search_idx` or similar